server/src/core/distribution/distribution-service.ts

```diff
@@ -1,10 +1,14 @@
 import { Db, Collection } from 'mongodb';
-import { DonationDistributionService, DonationDistributionResults, DonationDistributionArgs } from './types';
+import { DonationDistributionService, DonationDistributionResults, DonationDistributionArgs, BeneficiaryFilter } from './types';
 import { runDonationDistribution } from './run-distribution';
 import { generateId } from '../util';
 import { AppError, createAppError } from '../error';
 
 const COLLECTION = 'donation_distributions';
+export const arbitraryFilter: BeneficiaryFilter = { 
+  isVetted: { $exists: false }, 
+  beneficiaryStatus: { $exists: false }
```

Comment:
Please remove these constraints from the default filters. We should not change the way the current distribution works, that means we should not add any new constraints to the default case. It should just be an empty filter object.

Furthermore, `isVetted: {$exists: false}` is not the opposite of `isVetted: true`. `isVetted: false` is also unvetted. So the correct query for unvetted beneficiaries is `isVetted: {$ne: true}` and the correct query for unverified beneficiaries is `beneficiaryStatus: {$ne: 'verified'}`

Reply:
The existing distribution should apply to all beneficiaries, whether vetted or unvetted.

The new vetted distribution should only apply to vetted verified beneficiaries.